Lib/defconQt/fontView.py

```diff
@@ -901,31 +962,32 @@ def _set_sortDescriptor(self, desc):
                         glyphs.append(glyph)
         else:
             glyphs = [self._font[k] for k in self._font.unicodeData
-                .sortGlyphNames(self._font.keys(), desc)]
+                      .sortGlyphNames(self._font.keys(), desc)]
         self.collectionWidget.glyphs = glyphs
         self._sortDescriptor = desc
 
     sortDescriptor = property(_get_sortDescriptor, _set_sortDescriptor,
-        doc="The sortDescriptor. Takes glyphs from the font and sorts them \
-        when set.")
+                              doc="The sortDescriptor. Takes glyphs from the "
+                              "font and sorts them when set.")
```

Comment:
Somehow making a new string like this concats to the previous one? Did not know.